ee/clickhouse/queries/test/__snapshots__/test_trends.ambr

```diff
@@ -332,7 +332,7 @@
        AND timestamp <= toDateTime('2020-01-02 23:59:59')
        AND (has(['technology'], replaceRegexpAll(JSONExtractRaw(group0_properties, 'industry'), '^"|"$', ''))) )
   GROUP BY actor_id
-  LIMIT 200
+  LIMIT 100
```

Comment:
Why is this here...? Feels like the diff I was seeing from one of my unmerged PRs...

Reply:
I didn't do that `Update snapshots` commit? wtf?

Reply:
I had to nuke my laptop a few days ago, and reinstall everything, checking what happened with `black` 🤦

Reply:
Sorry about the snapshots, my PR went in, didn't update this, but then master started failing, so a subsequent PR fixed this. You were in a race condition as you started your PR after the first one went in, but before the second one did 😂